Earthfile

```diff
@@ -115,7 +115,7 @@ test-integration:
   COPY +go/translate /translate
   COPY --dir migrate/mysql migrate
   WITH DOCKER --compose compose.yaml --service mysql --pull migrate/migrate:v$migrate_version --pull golang:$go_version-alpine
-    RUN --no-cache \
```

Comment:
If I understood correctly there is no point, in running integration tests with a disabled cache, as the token env variable is changed to `path_to_token` env variable

Drastically improves integration tests execution (only locally tho )

Reply:
Without `--no-cache`, the google account access json is cached in the buildkit layer. We require `--no-cache` for security reasons.

Reply:
https://github.com/translate-agent/translate/pull/134/files#diff-2705f6016a64189abd14041e2f3e8dbb829553295d9088e4eb42eccb5e61bf47R132